src/libraries/System.Security.Cryptography.X509Certificates/tests/ChainTests.cs

```diff
@@ -326,7 +348,7 @@ public enum BuildChainCustomTrustStoreTestArguments : int
         }
 
         [Fact]
-        public static void BuildChainWithSystemTrustAndCustomTrustCertificates()
+        public static void BuildChainWithSBystemTrustAndCustomTrustCertificates()
```

Comment:
```suggestion
        public static void BuildChainWithSystemTrustAndCustomTrustCertificates()
```

I don't think we need the extra B :)